workflows/preprocessing.nf

```diff
@@ -138,7 +138,10 @@ workflow PREPROCESSING {
         if (genomes && genomes[meta.genome]){
             meta = meta + ["genome": genomes[meta.genome]]
         }
-
+        // set the aligner
+        if (aligner && !meta.aligner) {
+            meta = meta - meta.subMap("aligner") + ["aligner": aligner]
```

Comment:
```suggestion
            meta = meta + ["aligner": aligner]
```
This automatically overwrites the `aligner` meta option (but your solution was fine too :p)

Reply:
aha, didn't know that 😄